tests/base/src/test/java/org/keycloak/tests/admin/authz/fgap/UserResourceTypeEvaluationTest.java

```diff
@@ -60,10 +60,17 @@
 @KeycloakIntegrationTest(config = KeycloakAdminPermissionsServerConfig.class)
 public class UserResourceTypeEvaluationTest extends AbstractPermissionTest {
 
-    @InjectUser(ref = "alice")
+    @InjectUser(ref = "alice", realmRef = REALM_REF)
     ManagedUser userAlice;
 
-    @InjectAdminClient(mode = InjectAdminClient.Mode.MANAGED_REALM, client = "myclient", user = "myadmin")
+    @InjectAdminClient(
+            mode = InjectAdminClient.Mode.MANAGED_REALM,
+            realm = REALM_REF,
```

Comment:
In `InjectAdminClient` realm refers to the realm name, not the realm-ref.

Reply:
That is how I initially had it. However, we decided that the admin client and the factory shouldn't use the `getDependency` method as it can dynamically create new instances if no available instances are deployed.